website/docs/design-patterns/component-inheritance.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Component Inheritance Atmos Design Pattern
+sidebar_position: 8
+sidebar_label: Component Inheritance
+description: Component Inheritance Atmos Design Pattern
+---
+
+# Component Inheritance
+
+The **Component Inheritance** Design Pattern describes the mechanism of deriving Atmos components from one or more base components, allowing reusing
+the base components' configurations.
+
```

Comment:
could be useful to have a note about "derived components". Our refarch docs frequently refer to "derived" components, but is the same thing as an inherited component

this is a suggestion, feel free to tweak/move/remove of course

```suggestion
:::info Derived Components

Inherited Components are often referred to as Derived Components, which describe the same type of Atmos components.

:::